hdc/algo/ops/stats.py

```diff
@@ -217,9 +218,15 @@ def gammastd_yxt(
     if cal_stop is None:
         cal_stop = t
 
+    if nodata is None:
+        nodata = -9999
+
     for ri in range(r):
         for ci in range(c):
             xt = x[ri, ci, :]
+            if (xt != nodata).sum() == 0:
+                y[ri, ci, :] = -9999
```

Comment:
`= nodata` ?

Reply:
there are still a lot of -9999 constants around

Reply:
`gammastd` returns `-9999` as fill value. The idea is that `nodata` is used to determine input nodata, and output is fixed to `-9999`

Reply:
that's a confusing api then,  usually in/out nodata is the same or there are two options for src/dst. At the very least output nodata value should then be placed into a variable with reasonable name.